.gitignore

```diff
@@ -30,3 +30,17 @@
 *.exe
 *.out
 *.app
+
+# Auxiliar build files
+CMakeFiles
+cmake_install.cmake
+CMakeCache.txt
+Makefile
+install_manifest.txt
+pyuast.py
```

Comment:
I don't think this .py and python related files are needed, since they should not appear here.